### PR TITLE
Add reset settings

### LINKED ISSUE #487: core: frontend: Add settings menu
Add settings menu for user configuration of the interface

### LINKED ISSUE #542: core: Add option to reset settings
Delete entire .config folder and restart companion, doing everything from scratch

### PR DESCRIPTION
**Pull request by patrickelectric**
- I'm going to open an issue to have a better UX related to system restart after settings reset, once this PR get merged. 

docker: patrickelectric/companion-core:reset_settings

Fix #487
Fix #542

### CODE REVIEW

File: core/services/commander/main.py
Comment (rafaellehmkuhl) [inline core/services/commander/main.py:75]: `async def reset(`

File: core/services/commander/main.py
Comment (rafaellehmkuhl) [inline core/services/commander/main.py:98]: I would add a generic "Could not reset user configuration" before the specific error, so it becomes more clear what's happening.

File: core/frontend/src/components/app/SettingsMenu.vue
Comment (rafaellehmkuhl) [inline core/frontend/src/components/app/SettingsMenu.vue:94]: "No details available."

File: core/frontend/src/components/app/SettingsMenu.vue
Comment (rafaellehmkuhl) [inline core/frontend/src/components/app/SettingsMenu.vue:87]: This variable seems to be a perfect fit for a confirmation dialog, although this should not block the PR.

File: core/frontend/src/components/app/SettingsMenu.vue
Comment (patrickelectric) [reply core/frontend/src/components/app/SettingsMenu.vue:94]: Done.

File: core/frontend/src/components/app/SettingsMenu.vue
Comment (patrickelectric) [reply core/frontend/src/components/app/SettingsMenu.vue:87]: This variable is to avoid people using this REST API without confirming that they know what they are doing.

Comment (patrickelectric) [reply core/frontend/src/components/app/SettingsMenu.vue:87]: Yes, I agree, this will be on the issue after this PR get merged.

**Review by rafaellehmkuhl — APPROVED** [review]

**Pull request merged**
